portfolio/src/main/webapp/quiz.js

```diff
@@ -0,0 +1,62 @@
+function makeQuiz() {
+  questions.forEach((currentQuestion, questionNumber) => {
+    output.push(
+      `<input type="checkbox" id="question${questionNumber}" name="question${questionNumber}" 
+            value="yes">
+            <label for="question${questionNumber}">${currentQuestion}</label></br>`
+    );
+  });
+  quizContainer.innerHTML = output.join("");
+}
+function showResults() {
+  let count = 0;
+  const resultHeader = document.getElementById("result-header");
+  resultHeader.style = "visibility:visible;";
+  questions.forEach((currentQuestion, questionNumber) => {
+    if (document.getElementById(`question${questionNumber}`).checked === true) {
+      count++;
+    }
+  });
+  document.getElementById(
+    "score"
+  ).textContent = `Congratulations! Your score was ${count}.`;
+
+  if (count <= 1) {
+    document.getElementById("score-comment").textContent =
+      "You are definitely not a cat.";
+  } else if (count <= 7) {
+    document.getElementById("score-comment").textContent =
+      "You are not a cat, but you do have some catlike tendencies.";
+  } else if (count <= 10) {
+    document.getElementById("score-comment").textContent =
+      "You might be a cat or some other animal.";
+  } else {
+    document.getElementById("score-comment").textContent =
+      "Meow. You are a cat.";
+  }
+  window.scrollBy(000, 300);
```

Comment:
Consider https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView, which scrolls the element you want the user to see into view.

Scrollby might overshoot the target element on wide screens, and undershoot on narrow ones.

Reply:
done!